addons/elasticsearch/elasticsearch.yaml

```
@@ -10,9 +10,9 @@ metadata:
     # on the cluster, this hack will trigger re-queue on Addons until one exists.
     kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "6.8.2-7"
-    appversion.kubeaddons.mesosphere.io/elasticsearch: "6.8.2"
-    values.chart.helm.kubeaddons.mesosphere.io/elasticsearch: "https://raw.githubusercontent.com/helm/charts/6bfbc8018cd4440637b07c7559d5812e4d9db34d/stable/elasticsearch/values.yaml"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "6.8.12-1"
+    appversion.kubeaddons.mesosphere.io/elasticsearch: "6.8.12"
+    values.chart.helm.kubeaddons.mesosphere.io/elasticsearch: "https://raw.githubusercontent.com/elastic/helm-charts/e4ab721d108439a23187dc4da674ac20e6ede056/elasticsearch/values.yaml"
```

```suggestion
    values.chart.helm.kubeaddons.mesosphere.io/elasticsearch: "https://raw.githubusercontent.com/elastic/helm-charts/74891dda8864cbe7be8bc296db85763a77028ac7/elasticsearch/values.yaml"
```
Sorry my bad, was just working on migrating kibana to elastic charts and realized this is the correct sha (I had grabbed the latest commit earlier but this should be the correct one off 6.8.12 release)